README.md

```diff
@@ -2,144 +2,10 @@
 
 Monokle Admission Controller is an admission controller for validating resources in the cluster.
 
-## Development
-
-### Prerequisites
-
-* Minikube (or any other K8s cluster running)
-* kubectl
-* Skaffold
-* nodejs
-
-### Running
-
-#### Minikube
-
-Start Minikube:
-
-```bash
-minikube start --uuid 00000000-0000-0000-0000-000000000001 --extra-config=apiserver.enable-admission-plugins=ValidatingAdmissionWebhook
-```
-
-#### Deploying
-
-```bash
-./scripts/deploy.sh
-```
-
-Namespaced resources (webhook server) will be deployed to dedicated `monokle-admission-controller` namespace, to watch it you can run:
-
-```bash
-watch kubectl -n monokle-admission-controller get all,CustomResourceDefinition,ValidatingWebhookConfiguration,secrets
-```
-
-After it runs, the result should be something like:
-
-```bash
-NAME                                                       READY   STATUS    RESTARTS   AGE
-pod/monokle-admission-controller-server-6958c9bbf8-jvkvk   1/1     Running   0          5m11s
-
-NAME                                          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
-service/monokle-admission-controller-server   ClusterIP   10.99.122.106   <none>        443/TCP   5m11s
-
-NAME                                                  READY   UP-TO-DATE   AVAILABLE   AGE
-deployment.apps/monokle-admission-controller-server   1/1     1            1           5m11s
-
-NAME                                                             DESIRED   CURRENT   READY   AGE
-replicaset.apps/monokle-admission-controller-server-6958c9bbf8   1         1         1       5m11s
-
-NAME                                                                      CREATED AT
-customresourcedefinition.apiextensions.k8s.io/policies.monokle.io         2023-10-12T12:16:04Z
-customresourcedefinition.apiextensions.k8s.io/policybindings.monokle.io   2023-10-12T12:16:04Z
-
-NAME                                                                                               WEBHOOKS   AGE
-validatingwebhookconfiguration.admissionregistration.k8s.io/monokle-admission-controller-webhook   1          5m11s
-
-NAME                                      TYPE                                  DATA   AGE
-secret/default-token-w56nz                kubernetes.io/service-account-token   3      5m39s
-secret/monokle-admission-controller-tls   kubernetes.io/tls                     2      5m1s
-secret/monokle-policies-sa-token-fcpld    kubernetes.io/service-account-token   3      5m49s
-```
-
-For getting info about CRDs:
+## Installation
 
-```bash
-kubectl get crd
-kubectl describe crd policies.monokle.io
-kubectl describe crd policybindings.monokle.io
-```
+// TBD
```

Comment:
Either remove or write this.

Reply:
I already moved it and reworked as part of #9 PR (see https://github.com/kubeshop/monokle-admission-controller/blob/f1ames/feat/release-flow/CONTRIBUTING.md). It's fixed there so I'll skip fixing it here.